Configuration/PyReleaseValidation/python/MatrixInjector.py

```diff
@@ -71,7 +71,7 @@ def __init__(self,opt,mode='init',options=''):
         if not self.wmagent:
             if not opt.testbed :
                 self.wmagent = 'cmsweb.cern.ch'
-                self.DbsUrl = "https://"+self.wmagent+"/dbs/prod/global/DBSReader"
+                self.DbsUrl = "https://cmsweb-prod.cern.ch/dbs/prod/global/DBSReader"
```

Comment:
although it was already hardcoded, why not configure it via env variable. This will help in future when we have to change it again.

Reply:
That would be the most flexible way, however it would then depend on each user having to `export` it before every use. Unless you are talking about env variables that are set by `cmsenv`, then yes, it could be set there. I also don't expect this to change that often that it would cause too much of inconvenience to change the hardcoded value.

Reply:
@justinasr , I mean use the env variable to override the default. If env variable is set then we use it otherwise fallback to default ( hardcoded in the release).

Reply:
@smuzaffar sure, should the whole url be configurable or just the host? Would these be appropriate names for the env variables: `DBSREADER_URL` (if the whole url is configurable) or `DBSREADER_HOST` (if only host is configurable)?

Reply:
@justinasr , I would suggest to make whole url configurable. About the name, I would suggest to use `CMS_DBSREADER_URL` . If you want, you can also add command-line option e.g. `--dbs-url ful-dbs-url` to `runTheMatrix.py` :-)